### DISCUSSION TITLE
Logo Design

### DISCUSSION THREAD

**Discussion by YuriDevAT** [YurisCodingClub/dev-cv#27] [Ideas]
How should the Logo of the app look like?
Let's collect some ideas (maybe from [Behance](https://www.behance.net/) or [dribbble](https://dribbble.com/)) and choose with which kind of design we want to go further.

- [ ] Design for the Logo
- [ ] Depending on the Logo, design for an Icon

**Comment by YuriDevAT** [comment]
I do like these kind of style of logo (gradient) and the combination with the name:

[See the whole logo design project](https://www.behance.net/gallery/153535893/Logo-Logo-design-Branding?tracking_source=search_projects%7Clogo+design)
<img width="559" alt="Screenshot 2022-10-15 at 10 03 48" src="https://user-images.githubusercontent.com/54622834/195976347-a76a6671-9773-42a5-921a-66421f463e66.png">

[See the whole logo design project](https://www.behance.net/gallery/154653825/Tech-Logo-Brand-Identity-Logo-Design?tracking_source=search_projects%7Clogo+design)
<img width="1077" alt="Screenshot 2022-10-15 at 10 04 14" src="https://user-images.githubusercontent.com/54622834/195976365-7a80231e-a865-43e4-a94f-26f3a6063017.png">

[See the whole logo design project](https://dribbble.com/shots/19299319-logo-logo-design-branding-logotype-brand-identity) 
<img width="982" alt="Screenshot 2022-10-15 at 10 03 00" src="https://user-images.githubusercontent.com/54622834/195976292-b63b5275-f427-4404-ab57-ee62eea52a00.png">

**Reply by MusfiqDehan** [reply]
Hey @YuriDevAT ,

What do you think about this design?

![image](https://user-images.githubusercontent.com/47440165/196031092-6b9b01cf-2294-4cc1-9ad0-0fc53a3790d3.png)

**Reply by YuriDevAT** [reply]
> Hey @YuriDevAT ,
> 
> 
> 
> What do you think about this design?
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/47440165/196031092-6b9b01cf-2294-4cc1-9ad0-0fc53a3790d3.png)
> 
> 

Thanks for your suggestion. Well, the Logo itself looks more like an icon copied from a library and nothing like the ones from the examples. It should be kind of unique. The logo should speak for itself and have brand recognition, even without the app name.
For the text, I think it's best when it is one line without space, maybe lowercase, camelcase, or similiar.

**Comment by sowmiyeh** [comment]
Hi @YuriDevAT 

I've completed the design and it would be great if you can share your feedback on the same so that I can create the pull request

![Dev CV - logo](https://user-images.githubusercontent.com/74345706/196041274-834c342e-0b5a-409c-971e-26c49d02b9ad.png)

**Reply by YuriDevAT** [reply]
Thank you for your suggestion 👍🏽  . Especially the Logo on the bottom right seems like a great fit for the app. I assume that the outer blue would be the color of e.g. the whole header itself and the logo/icon would be the white circle with the detailed logo in it?

When open a PR, please provide information on the HEX as well as the font. I am not sure if we should go with the font but I will try it out and do some accessibility checks (so the font would not be part of the svg I assume)

**Reply by sowmiyeh** [reply]
> Thank you for your suggestion 👍🏽 . Especially the Logo on the bottom right seems like a great fit for the app. I assume that the outer blue would be the color of e.g. the whole header itself and the logo/icon would be the white circle with the detailed logo in it?
> 
> When open a PR, please provide information on the HEX as well as the font. I am not sure if we should go with the font but I will try it out and do some accessibility checks (so the font would not be part of the svg I assume)

Thank you so much @YuriDevAT !

Yes the right side bottom is the logo, and the blue colour is the whole BG.

And then the typo was done by me, it was not taken from anywhere, of it is not needed we can exclude that.

And yes, please check my figma file for further details of the favicon - https://www.figma.com/file/qtNIexlS4i6fn1BXFMrFhr/Dev-cv-logo, I'll create a pull request after confirmation from you.

**Reply by YuriDevAT** [reply]
Thanks for sharing the Figma link. I made a comment in the file, suggesting using the bottom right as Logo and Favicon. I took the HEX Color out of your files (#311ABE for the blue), thanks.
